src/modules/compliance-validator/ComplianceValidator.ts

```
@@ -0,0 +1,263 @@
+  }
+
+  private async checkIndustrySpecificCompliance(
+    content: ParsedContent,
+    domain: string
+  ): Promise<ComplianceViolation[]> {
+    const violations: ComplianceViolation[] = [];
+
+    switch (domain) {
+      case 'healthcare':
+        const hipaaViolations =
+          await this.hipaaChecker.checkCompliance(content);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)